Middleware/checkAuthStatus.js

```diff
@@ -0,0 +1,28 @@
+import jwt from 'jsonwebtoken';
+import User from '../Models/users.js';
+
+const checkAuthStatus = async (req, res, next) => {
```

Comment:
nice middleware function. You could make sure that it returns the status code 401 unauthorized if the token cannot be varified.

Reply:
Done 👍